csharp/src/Microsoft.ML.OnnxRuntime/Microsoft.ML.OnnxRuntime.targets

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <ItemGroup>
+    <None Include="$(MSBuildThisFileDirectory)..\runtimes\win10-x64\native\onnxruntime.dll">
```

Comment:
I think I've discussed this before but, why are we using the `win10-x64` RID? Does the OnnxRuntime ONLY work on Windows 10 (and Windows Server 2016) machines?

Instead, I think the RID should be `win-x64`, meaning it can run on other Windows machines (Windows 7, 8.1, etc)

Reply:
This is pending on testing on other windows platforms. Currently onnxruntime.dll is tested on windows 10.